Wikipedia/Code/UIViewController+WMFOpenExternalUrl.m

```diff
@@ -88,8 +88,8 @@ - (void)wmf_presentExternalUrlAsSFSafari:(NSURL *)url {
     [self presentViewController:[[SFSafariViewController alloc] initWithURL:url] animated:YES completion:nil];
 }
 
-- (BOOL)isPartnerInfoConfigValid:(WMFZeroMessage *)msg {
-    return msg.partnerInfoText != nil && msg.partnerInfoUrl != nil;
+- (BOOL)isPartnerInfoConfigValid:(WMFZeroConfiguration *)config {
```

Comment:
Can we move this to a method on `WMFZeroConfiguration` instead? Its functionality fits better there than being a `NSURL` method in the `WMFOpenExternalURL` category